src/providers/GitHub/index.ts

```diff
@@ -172,9 +172,15 @@ export class GitHubDataProvider implements DataProvider<GitHubOptions> {
           file_sha: file.sha,
         });
 
+        // Decode the content blob as it is encoded
+        const decodedContent = Buffer.from(blob.data.content, 'base64').toString('utf8');
+
```

Comment:
This may output large amounts of non-text data if binaries are contained in the repo. Is that okay? Should we add UTF-8 detection?